src/config/index.ts

```diff
@@ -25,6 +25,6 @@ export const config = () => ({
 
     rabbitmq: {
         url: [process.env.MQ_URL ?? `amqp://user:bitnami@${process.env.MQ_HOST ?? 'localhost'}:5672`],
-        queue: process.env.MQ_QUEUE ?? 'api-consume-dev',
+        queue: process.env.MQ_QUEUE ?? 'api-aggregator-incoming',
```

Comment:
Would probably call it `api-queue` as the request handled by the api are not strictly exclusive to the aggregator. Also the api can receive events(without responses) as well as request(with responses) from the queue.